fmf/utils.py

```diff
@@ -532,11 +534,19 @@ def fetch(url, ref='master'):
     directory = url.replace('/', '_')
     destination = os.path.join(fmf_cache, directory)
 
+    # Use the default branch from origin of no ref provided
+    if ref is None:
+        ref = 'DEFAULT'
```

Comment:
```suggestion
        ref = '__DEFAULT__'
```

Reply:
Not sure how often one has branch DEFAULT but __DEFAULT__ seems to be more rare.

Reply:
Yeah, let's minimize possible conflict.